apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -164,8 +168,15 @@ defmodule Aecore.Chain.Worker do
       "Added block ##{new_block.header.height} with hash #{Base.encode16(new_block_hash)}, total tokens: #{inspect(total_tokens)}"
     end)
 
-    ## Store new block to disk
-    Persistence.write_block_by_hash(new_block)
+    ## Store new block to RocksDB
```

Comment:
can you combine all persistence to happen for a added block in one function inside the persistence worker? 
inside you can have the L171 - L178 as it is now.

Reply:
I will consider this, and will let you know, thanks